media-tv/plex-media-server/plex-media-server-1.23.1.4571.ebuild

```
@@ -0,0 +1,80 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit systemd unpacker
+
+DESCRIPTION="A free media library that is intended for use with a plex client"
+HOMEPAGE="https://www.plex.tv/"
+
+MY_COMMIT="6119e8eed"
+MY_PV="${PV}-${MY_COMMIT}"
+
+MY_URI="https://downloads.plex.tv/plex-media-server-new"
+SRC_URI="
+	amd64? ( ${MY_URI}/${MY_PV}/debian/plexmediaserver_${MY_PV}_amd64.deb )
+	x86? ( ${MY_URI}/${MY_PV}/debian/plexmediaserver_${MY_PV}_i386.deb )"
+
+LICENSE="Plex"
+SLOT="0"
+KEYWORDS="-* ~amd64 ~x86"
+RESTRICT="mirror bindist strip"
```

```suggestion
RESTRICT="bindist mirror"
```
I find adding `strip` to binary-only packages to be a poor trend I see across many packages. As I see it, this is for if you want to keep symbols (or if stripping breaks things) and not because it happens to be pre-stripped.

`QA_PREBUILT` is already there to prevent QA warnings regarding pre-stripped files.

Nevermind if you know of actual issues with stripping.